docs/source/documentation/client_configuration.rst

```diff
@@ -0,0 +1,115 @@
+How-to configure Substra clients
+================================
+
+Whether you are using SubstraFL or directly the Substra SDK, you need to configure one ``Client`` by organisation,
+in order to register the datasets and the functions you want to use.
+
+This how-to guide exposes the different options you have to configure your clients. It targets both first-time and
+advanced Substra users.
```

Comment:
What do you think to warn here on the order of priority ? IMO, this is quite error prone so we prefer to say it too many times than the opposite :)